browser/brave_wallet/sources.gni

```diff
@@ -13,15 +13,34 @@ brave_browser_wallet_deps = []
 
 if (brave_wallet_enabled) {
   brave_browser_wallet_sources += [
-    "//brave/browser/brave_wallet/brave_wallet_context_utils.cc",
-    "//brave/browser/brave_wallet/brave_wallet_context_utils.h",
+    "//brave/browser/brave_wallet/crypto_wallets_infobar_delegate.cc",
```

Comment:
these are the only files with circular deps so we put the rest in a separate target to better manage deps